source/Missions/Services/Agents/Messages/NetworkAgentDamage.cs

```diff
@@ -21,7 +24,7 @@ public class AgentDamageData : INetworkEvent
         [ProtoMember(4)]
         public AttackCollisionData AttackCollisionData { get; }
 
-        public AgentDamageData(
+        public NetworkAgentDamage(
```

Comment:
All events should be past-tense
```suggestion
        public NetworkAgentDamaged(
```